src/libmongoc/CMakeLists.txt

```diff
@@ -338,7 +338,10 @@ if (ENABLE_SHM_COUNTERS STREQUAL "ON")
          "Shared memory performance counters not supported on Mac or Windows")
    endif ()
    set (MONGOC_ENABLE_SHM_COUNTERS 1)
-   set (SHM_LIBRARIES rt)
+   find_library(RT_LIBRARY rt)
+   if (RT_LIBRARY)
+   	set (SHM_LIBRARIES rt)
+   endif ()
```

Comment:
Just to double check, librt is not necessary for `shm_open` and `shm_unlink` (which are instead part of libc). So if we cannot find librt, that is not an issue for BSD. Is that correct?

Reply:
Yes